cli-output/src/cli_output.rs

```diff
@@ -343,18 +343,24 @@ impl fmt::Display for CliValidators {
             validator: &CliValidator,
             total_active_stake: u64,
             use_lamports_unit: bool,
+            highest_last_vote: u64,
+            highest_root: u64,
         ) -> fmt::Result {
-            fn non_zero_or_dash(v: u64) -> String {
+            fn non_zero_or_dash(v: u64, max_v: u64) -> String {
                 if v == 0 {
-                    "-".into()
+                    "-         ".into()
+                } else if v == max_v {
+                    format!("{:>8}      ", v)
+                } else if v > max_v.saturating_sub(100) {
```

Comment:
Some reason this should be 100 and not DELINQUENT_VALIDATOR_SLOT_DISTANCE (128)?

Reply:
I'm was trying to avoid making the output even wider than it already so my goal was to stay with this template: `(-XX)`